salt/config/__init__.py

```diff
@@ -1579,7 +1579,7 @@ def _gather_buffer_space():
     'sign_pub_messages': True,
     'add_proxymodule_to_opts': False,
     'proxy_merge_grains_in_module': True,
-    'append_minionid_config_dirs': ['cachedir', 'pidfile'],
+    'append_minionid_config_dirs': ['cachedir', 'pidfile', 'default_include', 'include'],
```

Comment:
I think that if we have to distinguish between the caching, pidfile etc, we could also distinguish between the location of the `_schedule.conf` files and others that have to be per proxy rather than ons single file shared by all minions?.

At the same time, this raises other questions:

- what if you want a `.conf` file for all proxies?
- is it obvious enough that configuring as `default_include: proxy.d/*.conf` will look into `proxy.d/<proxy ID>/*.conf`?